cms/cms.go

```diff
@@ -132,21 +122,8 @@ func mkdir(permissions os.FileMode, dirs ...string) (err error) {
 	return
 }
 
-/* Create directories. Return a passive one (i.e. symlink does not point to it) */
-func createDirectories(assets string) (inactive string, err error) {
-	if err = mkdir(0755, assets, "a", "banners"); err != nil {
-		return
-	}
-	if err = mkdir(0755, assets, "a", "persons"); err != nil {
-		return
-	}
-	if err = mkdir(0755, assets, "b", "banners"); err != nil {
-		return
-	}
-	if err = mkdir(0755, assets, "b", "persons"); err != nil {
-		return
-	}
-
+/* Create/clear directories. Return a passive one (i.e. symlink does not point to it) */
```

Comment:
consider creating a new fresh directory each time instead of a-b.

Reply:
done

Reply:
done